### ISSUE TITLE
increase z-index to 999999 because #wpadminbar z-index is 99999.

### ISSUE THREAD

**Issue by jasomdotnet** [englishextra/iframe-lightbox#14]
Increase z-index to 999999 because #wpadminbar z-index is 99999.

**Comment by englishextra** [comment]
Good catch

seems to be an unofficial standard:

```css
.glightbox-container {
	width: 100%;
	height: 100%;
	position: fixed;
	top: 0;
	left: 0;
	z-index: 99999 !important;
```
and yes

```
#wpadminbar {
  direction: ltr;
  color: #CCC;
  font: 400 13px/32px "Open Sans",sans-serif;
  height: 32px;
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  min-width: 600px;
  z-index: 99999;
```

Ok will fix thanks

**Comment by englishextra** [comment]
This is fixed in v0.2.0
